docs/book/src/developer/providers/migrations/v1.10-to-v1.11.md

```diff
@@ -612,3 +633,28 @@ import (
 
 - the `conditions` package provides access to utils for managing `metav1.Conditions` in `status.conditions`
 - the `patch` package provides access to utils for patching objects in this phase
+
+Important!
+- Please pay special attention to use `sigs.k8s.io/cluster-api/util/patch` import everywhere,
+  because if by mistake you use `sigs.k8s.io/cluster-api/util/deprecated/v1beta1/patch` in some files this won't lead to compile errors
+  but conditions are going to be dropped at runtime.
```

Comment:
```suggestion
  because using `sigs.k8s.io/cluster-api/util/deprecated/v1beta1/patch` by mistake could in some cases lead to dropping conditions at runtime,
  while not having compile errors.
```